assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
+}
+p {
+  font-size: 1.25em;
+  color: gray;
+  line-height: 32px;
+}
+
+h1 {
+  font-size: 40px !important;
+  color: black;
+}
+
+.white-text p{
+  color: white;
+}
+.white-text h1{
+  color: white;
+}
+
+.divider-text {
+  padding: 40px 50px 50px 50px;
+}
+
+/* Menu Logo */
+
+h1.top-logo{
+  background: url("../img/logo.png") no-repeat 0 0;
+  margin-left: 18px;
+}
+h1.top-logo a{
+  display:block;
+  width: 200px;
+  text-indent:-9999px;
+}
+
+@media(max-width: 768px) {
+  h1.top-logo{
+    background: url("../img/logo-mobile.png") no-repeat 0 0;
+    margin-left: 18px;
+  }
+
+  h1.top-logo a{
+    display:block;
+    width: 50px;
+    text-indent:-9999px;
+  }
+}
+
+/* Menu Styling */
+#Menu {
+
+}
+
+#Menu ul{
+  list-style: none;
+  top: 13px;
+  right: 0;
+  position: absolute;
+  display: -webkit-inline-box;
+  float: right;
+  padding: 15px 25px 0 0;
+}
+#Menu ul li{
+  padding: 0 15px 0 0;
+}
+
+/*body styling */
+
+body{
+  margin-left: 0;
+  margin-right: 0;
+
+}
+/*full width image boxes*/
+.image-box {
+  height: 600px;
+}
+
+.first-box {
```

Comment:
What happens if you decide you want to move the `second-box` above the `first-box` within the HTML? Do those class names still make sense?

You might want to consider having a class referencing the specific image that the style applies, like `watch-img` or `pencil-img`. That way you won't ever be confused about what background will be applied.